### ISSUE TITLE
feat: add (hidden) skip to main content button

### ISSUE THREAD

**Issue by hollandjg** [thepolicylab-projectportals/project-portal-theme#281]
- should be very first tababble thing
- should jump to anchor for the main content
- check the interwebs for how to do this from an accessibility perspective
- doesn't need to be a visible button

Questions: Which aria tag do we need for this?

From https://github.com/thepolicylab/example-project-portal/issues/162
